map-example/map-example.cabal

```diff
@@ -19,12 +19,12 @@ source-repository head
   type:                git
   location:            https://github.com/kowainik/containers-backpack.git
 
-library
-  hs-source-dirs:      src
-  exposed-modules:     Dummy
-  build-depends:       base, map-sig, map-containers, map-contrib
-  mixins:              map-contrib (Map.Contrib.Dummy as Map.Contrib.Dummy.IntMap)
+executable map-exe
+  hs-source-dirs:      app
+  main-is:             Main.hs
+  build-depends:       base, map-containers-intmap, map-contrib
+  mixins:              map-contrib (Map.Contrib.Group as Map.Contrib.Group.IntMap)
                           requires (Map as Map.IntMap)
```

Comment:
I think it's a good idea to use all our implementations in the example. With this we will be sure that all our signatures are implemented properly. For example, `map-unordered` doesn't implement `fromList` function and this is not checked anywhere. 

You can look at this file to see how to use multiple mixins:

* https://github.com/ezyang/backpack-regex-example/blob/2444a5084dcc6f82fcab9209e769fac86a426d79/regex-example/regex-example.cabal#L9-L12